### PR TITLE
Add PRR survey to OIDC Discovery KEP

### PR DESCRIPTION
**Pull request by mtaufen**
Add PRR survey as part of preparing to move to GA.

### CODE REVIEW

File: keps/sig-auth/1393-oidc-discovery/README.md [hunk 1]
```diff
@@ -296,11 +303,208 @@ of versioning. However, we can still treat graduation in terms of
...
+    - Details: API server logs, or ability of workloads to authenticate to
+      Relying Parties.
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  We expect the endpoints to maintain high reliability, with reliability
+  matching that of kube-apiserver.
+
+* **Are there any missing metrics that would be useful to have to improve observability
+of this feature?**
+  It would be nice to have usage metrics for this endpoint. We haven't added
```
Comment (johnbelamaric) [inline keps/sig-auth/1393-oidc-discovery/README.md:416]: Is there an existing issue to cover this? I am gathering that the issue is you are serving up a static resource and so any metrics plumbing would have to be in the basic HTTP server in the api server. You don't have any code hit after generating the static data, so you have nowhere to hook in a metric?

If that's the case, perhaps you can at least create an issue and get it on the radar of api machinery, if it's not already.

**Review by johnbelamaric — COMMENTED** [review]
/approve
/lgtm

File: keps/sig-auth/1393-oidc-discovery/README.md (hunk 1 above)
Comment (mtaufen) [reply keps/sig-auth/1393-oidc-discovery/README.md:416]: Yeah more or less. Just registering a route doesn't seem to get you metrics by default. I could probably add some explicitly in the route handler, but a more transparent solution (with consistent route -> metric name mapping) would be preferable. 

Tracking issue: https://github.com/kubernetes/kubernetes/issues/98585

**Pull request merged**